src/NzbDrone.Core/ImportLists/ImportListType.cs

```diff
@@ -6,6 +6,7 @@ public enum ImportListType
         TMDB,
         Trakt,
         Other,
-        Advanced
+        Advanced,
+        Plex
```

Comment:
Given these are just used for ordering, We should likely move Plex up above Advanced.

Reply:
Looking at this now I wonder if moving it above `Other` makes sense. Keep the first-party lists together, the random lists together, and then the advanced lists at the bottom